doc/nrf/ug_zigbee_memory.rst

```diff
@@ -23,9 +23,10 @@ RAM and flash memory requirements
 
 RAM and flash memory requirement values differ depending on the DK, and the programmed sample.
 
-The following tables list memory requirement values (in KB) for Zigbee samples.
+The following tables list memory requirement values for Zigbee samples.
```

Comment:
I suggest using units to be crystal clear:
The following tables list memory requirement values (in B) for Zigbee samples.